### DISCUSSION TITLE
Any way to suppress the answer for custom alexa command?

### DISCUSSION THREAD

**Discussion by Kurisudes** [alandtse/alexa_media_player#1630] [Q&A]
Hello all,

I am using the command below every night as my night story. Every night alexa answers "Playing Chris' audible Der kleine Bruder ..."
If my wife is already sleeping it wakes her up.
Is there anyway to suppress this kind of answer?

`service: media_player.play_media
data:
  media_content_type: custom
  media_content_id: play Der kleine Bruder on Audible
target:
  entity_id: media_player.echo_dot`

I already tried setting the volume to 0 but the volume of the answer doesn't change only the media is not audible.
I also found some command for muting which I tied. That didn't do anything.

I would be very happy, if somebody can help.

**Comment by Jazzik777** [comment]
Did anyone find an answer to this?

I've done the same with volume and it just turns it back up to 1 to say the reply.

**Comment by Kurisudes** [comment]
> Did anyone find an answer to this?
> 
> 
> 
> I've done the same with volume and it just turns it back up to 1 to say the reply.

I still have no solve for this, unfortunately.

**Reply by giveyouup** [reply]
I have a solution! Bought a dedicated alexa device then opened it up and snipped the speaker wire.
